### PR TITLE
responsive home page

### PR DESCRIPTION
**Pull request by E-fais**
## ✔️ Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

**Comment by E-fais** [comment]
Fix issue #21

File: index.html [hunk 1]
```diff
@@ -8,26 +8,31 @@
     <meta name="description" content="Pradumna Saraf Portfolio">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="style.css">
+    <script src="main.js"></script>
```
Comment (Pradumnasaraf) [inline index.html:11]:
```suggestion

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, thank you for the PR. I think we don't need the js file. Otherwise Looks good.

File: index.html (hunk 1 above)
Comment (E-fais) [reply index.html:11]: Yes ;)

**Comment by E-fais** [comment]
Done ✔️

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by E-fais** [comment]
Thanks for accepting my PR .

**Pull request merged**